CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `ledger_index` optional param for all the main account methods
 - `TicketCreate` transaction model
+- `UnknownRequest` model for unsupported request types
```

Comment:
i don't love `CustomRequest` because this is _also_ being used by `Request.from_dict` if the request type cannot be determined. in that case, it's not really a custom request, it's more of an unknown request.

feels like we can have a name that captures both. `GenericRequest`?

Reply:
I can do `GenericRequest`